301-eventhub-namespace-vnet/azuredeploy.json

```diff
@@ -8,16 +8,29 @@
 				"description": "Name of the Event Hubs namespace"
 			}
 		},
+		"vnetResourceGroupName": {
+			"type": "string",
+			"metadata": {
+				"description": "Name of the Virtual Network Resource Group"
+			}
+		},
 		"vnetRuleName": {
 			"type": "string",
 			"metadata": {
 				"description": "Name of the Virtual Network Rule"
 			}
 		},
-		"virtualNetworkSubnetId": {
+		"subnetName": {
+			"type": "string",
+			"metadata": {
+				"description": "Name of the Virtual Network Sub Net"
+			}
+		},
+		"Location":{
 			"type": "string",
+			"defaultValue": "South Central US",
```

Comment:
use resourceGroup().location for the defaultValue - then override via parameter file

Reply:
resolved